apps/ui/components/reservation/ReservationCancellation.tsx

```diff
@@ -182,7 +217,9 @@ const ReservationCancellation = ({ id }: Props): JSX.Element => {
   }, [register]);
 
   const bylineContent = useMemo(() => {
-    const reservationUnit = reservation?.reservationUnits[0];
+    const reservationUnit = reservation?.reservationUnits
+      ? reservation?.reservationUnits[0]
+      : null;
```

Comment:
`reservation?.reservationUnits?.[0]` works also

Reply:
That's true! 😅